### DISCUSSION TITLE
A little help for Windows users

### DISCUSSION THREAD

**Discussion by batarela** [CastagnaIT/plugin.video.netflix#1607] [Ideas]
CastagnaIT in Windows does not work only the play option, everything else works. Can you add through the context menu the possibility of copying the name of the movie or series, so that we can more easily find the movie or series in the browser on the netflix page without typing (copy paste).

Thanks for your reply in advance

**Comment by Falke-Design** [comment]
@CastagnaIT if you give the ok, I would implement that

**Comment by CastagnaIT** [comment]
i accept new features that could have potential use for the majority of users
and avoid things that will be used by just a couple of users

more functionality == more time for maintenance
this addon is already much more complex of the majority of kodi video addons
and imo it is preferable to invest time on how to find a solution to the problem to play videos on these systems
so its not accepted

As i often i want remember in these cases, since the source code is free, fork it and add your own custom additions
